Assets/Python/Stability.py

```diff
@@ -1538,13 +1533,12 @@ def getResurrectionCities(iPlayer, bFromCollapse = False):
 				if bCapital:
 					lFlippingCities.append(city)
 					
-		# if only up to two cities wouldn't flip, they flip as well (but at least one city has to flip already, else the respawn fails)
-		if len(lFlippingCities) + 2 >= len(lPotentialCities) and len(lFlippingCities) > 0 and len(lFlippingCities) * 2 >= len(lPotentialCities) and not bFromCollapse:
-			lFlippingCities = []
-			# cities in core are not affected by this
-			for city in lPotentialCities:
-				if not city.plot().isCore(city.getOwner()):
-					lFlippingCities.append(city)
+	# if only up to two cities wouldn't flip, they flip as well (but at least one city has to flip already, else the respawn fails)
+	if len(lFlippingCities) + 2 >= len(lPotentialCities) and len(lFlippingCities) > 0 and len(lFlippingCities) * 2 >= len(lPotentialCities) and not bFromCollapse:
+		# cities in core are not affected by this
+		for city in lPotentialCities:
+			if not city.plot().isCore(city.getOwner()) and city not in lFlippingCities:
+				lFlippingCities.append(city)
 			

```

Comment:
This is the bugfix.

First, this part was inside the for-loop starting in line 493.

Secondly, if this part is executed, it was possible that some cities that were included in the first list didn't get included in the new lFlippingCities list. And the other way around. Now, the old list is maintained and is extended with new cities.